jerry-core/jrt/jrt.h

```diff
@@ -108,26 +108,14 @@ extern void __noreturn jerry_unimplemented (const char *, const char *, const ch
 #define JERRY_DDLOG(...) JERRY_LOG (2, __VA_ARGS__)
 #define JERRY_DDDLOG(...) JERRY_LOG (3, __VA_ARGS__)
 #else /* !JERRY_ENABLE_LOG */
-#define JERRY_DLOG(...) \
-  do \
-  { \
-    if (false) \
-    { \
-      jerry_ref_unused_variables (0, __VA_ARGS__); \
-    } \
-  } while (0)
-#define JERRY_DDLOG(...) JERRY_DLOG (__VA_ARGS__)
-#define JERRY_DDDLOG(...) JERRY_DLOG (__VA_ARGS__)
+#define JERRY_DLOG(...)
+#define JERRY_DDLOG(...)
+#define JERRY_DDDLOG(...)
 #endif /* !JERRY_ENABLE_LOG */
 
 #define JERRY_ERROR_MSG(...) jerry_port_errormsg (__VA_ARGS__)
 #define JERRY_WARNING_MSG(...) JERRY_ERROR_MSG (__VA_ARGS__)
 
-/**
- * Mark for unreachable points and unimplemented cases
- */
-template<typename... values> extern void jerry_ref_unused_variables (const values & ... unused);
```

Comment:
Hm, I think I understand you. In this case the easiest way is just remove the template expression from the jerry_ref_unused_variables.

Note:
  * Removed ECMA_BUILTIN_CP_UNIMPLEMENTED macros since every built-in are implemented.
  * When I ran a precommit I got the following messages:

[jerry-core/parser/regexp/re-compiler.cpp:53]: (style) A pointer can not be negative so it is either pointless or an error to check if it is not.
[jerry-core/parser/regexp/re-compiler.cpp:62]: (style) A pointer can not be negative so it is either pointless or an error to check if it is not.

The static cppchecker emits these errors. Do you have any ideas how to eliminate these?